includes/class-freemius.php

```diff
@@ -6514,6 +6514,29 @@ private function maybe_schedule_sync_cron() {
             $this->schedule_sync_cron();
         }
 
+        /**
+         * @author Leo Fajardo (@leorw)
+         * @since  2.9.1
+         */
+        private function maybe_schedule_install_sync_cron() {

```

Comment:
Let's also propagate all the arguments from the original method `$except_blog_id`